Symfony/CS/Tokenizer/Token.php

```diff
@@ -149,6 +145,25 @@ public function equalsAny(array $others, $caseSensitive = true)
     }
 
     /**
+     * A helper method used to find out whether or not a certain input token has to be case-sensitively matched.
+     *
+     * @param bool|bool[] $caseSensitive global case sensitiveness or an array of booleans, whose keys should match
+     *                                   the ones used in $others. If any is missing, the default case-sensitive
+     *                                   comparison is used.
+     * @param int|string  $key           the key of the token that has to be looked up
```

Comment:
So, is it enough to change the docblock to "int"? Should I also remove the tests using string keys?

BTW, the method is tested by the tests for equalsAny and findSequence. Do we need specific tests for it?